config/initializers/rack_middleware.rb

```diff
@@ -27,6 +27,11 @@
   # to the start of the Rack middleware chain.
   config.middleware.insert_before(0, ::Rack::DefaultContentType)
 
+  # If using Prometheus telemetry, we want to ensure that the middleware
+  # which collects and exports metrics is loaded near the start of the 
+  # middleware chain to prevent any modifications to the incoming requests
+  config.middleware.insert_before(1, Monitoring::Middleware::ConjurExporter)
```

Comment:
Why 1, I would have thought 0 ? Are we relying on `::Rack::DefaultContentType` ? 

```suggestion
  config.middleware.insert_before(0, Monitoring::Middleware::ConjurExporter)
```